R/loo_moment_matching.R

```diff
@@ -286,14 +286,20 @@ loo_moment_match_i <- function(i,
     # 1. match means
     trans <- shift(x, uparsi, lwi)
     # gather updated quantities
-    quantities_i <- update_quantities_i(x, trans$upars,  i = i,
+    quantities_i <- tryCatch(update_quantities_i(x, trans$upars,  i = i,
                                         orig_log_prob = orig_log_prob,
                                         log_prob_upars = log_prob_upars,
                                         log_lik_i_upars = log_lik_i_upars,
                                         r_eff_i = r_eff_i,
                                         cores = 1,
                                         is_method = is_method,
                                         ...)
+                        )
+    if (is_try_error(quantities_i)) {
```

Comment:
I don't think `is_try_error` is a base R function. Is this from a package? I think not finding this function is causing the unit test errors. If you use `try` then you can do

```suggestion
    if (inherits(quantities_i, "try-error")) {
```

Reply:
Oops, `is_try_error` is from `brms` and I forgot to run tests in fresh R session